submit_cgap/portal_network_access.py

```diff
@@ -0,0 +1,91 @@
+# This file contains centralized functions for all Portal interactions used by SubmitCGAP.
+
+import io
+import json
+import requests
+from typing import Callable, Optional, Tuple, Union
+from dcicutils import ff_utils
+from dcicutils.misc_utils import environ_bool, PRINT
+
+
+DEBUG_PROTOCOL = environ_bool("DEBUG_PROTOCOL", default=False)
+
+
+def portal_metadata_post(schema: str, data: dict, auth: Tuple) -> dict:
```

Comment:
These functions are a good idea but I think ultimately belong in ff_utils ie: ``ff_utils.debug_post_metadata``? What do you think @netsettler

Reply:
Having this dialog offline.

Reply:
I think let's leave them here for now since some of these call `ff_utils` and some call `requests`. I want to get that detail sorted out, but I think I'm leaning toward heavier use of `requests` and not having this call `ff_utils` at all if possible. I don't think for these situations we need `ff_utils`, though some adaptation may be needed. Or maybe I just want a cleaner tool in `ff_utils` that is no so cluttered with unwanted functionality. But all the more reason to just sit with this awhile here.